src/routes/docs/products/databases/collections/+page.markdoc

```diff
@@ -4,7 +4,8 @@ title: Collections
 description: Organize your data with Appwrite Collections. Explore how to create and configure collections to store and structure your data effectively.
 ---
 Appwrite uses collections as containers of documents. Each collection contains many documents identical in structure.
-The terms collections and documents are used because the Appwrite JSON REST API resembles the API of a traditional NoSQL database.
+While Appwrite uses SQL, the terms collections and documents are used because the Appwrite JSON REST API resembles
+the API of a traditional NoSQL database, making it intuitive and user-friendly.
```

Comment:
To make this flow better with the previous sentence, I focus on the terms first rather than saying "Appwrite uses SQL" first.

Reply:
Got it. How's the new update?